verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -24,7 +24,30 @@
 namespace verilog {
 namespace kythe {
 
-void KytheFactsExtractor::Visit(const IndexingFactNode& node) {
+void KytheFactsExtractor::ExtractKytheFacts(const IndexingFactNode& root) {
+  CreatePackageScopes(root);
+  IndexingFactNodeTagResolver(root);
+}
+
+void KytheFactsExtractor::CreatePackageScopes(const IndexingFactNode& root) {
+  // Searches for packages as a first pass and saves their scope to be used for
+  // imports.
+  for (const verible::VectorTree<IndexingNodeData>& child : root.Children()) {
```

Comment:
`const IndexingFactNode&`?  (you may use the `typedef`, it makes it obvious that the children are the same type as the object itself)

If you elect to change this, check elsewhere too for consistency.